test/tools/d_do_test.d

```diff
@@ -674,7 +674,7 @@ int tryMain(string[] args)
     Result testCombination(bool autoCompileImports, string argSet, size_t permuteIndex, string permutedArgs)
     {
         string test_app_dmd = test_app_dmd_base ~ to!string(permuteIndex) ~ envData.exe;
-
+        string command; // copy of the last executed command, s.t. it can be re-invoked on failures
```

Comment:
I would prefer if we didn't start using informal chatting abbreviations.  Please spell out "so that".